doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -93,11 +89,11 @@ The Reference Model requires the support of Huge Pages in `i.cap.018` which is s
+There are a number of different container runtimes. The simplest form, low-level container runtimes, just manage the OS capabilities such as cgroups and namespaces, and then run commands from within those cgroups and namesapces. An example of this type of runtime is runc, which underpins many of the higher-level runtimes and is considered a reference implementation of the [Open Container Initiative (OCI) runtime spec](https://github.com/opencontainers/runtime-spec). This specification includes detail on how an implementation (i.e. an actual container runtime such as runc) must, for example, configure resource shares and limits (e.g. CPU, Memory, IOPS) for the containers that Kubernetes (via the kubelet) schedules on that host. This is important in ensuring the features and capabilities described in [chapter 5 of the RM](../../../ref_model/chapters/chapter05.md) are supported by this RA and delivered by any downstream Reference Implementations (RIs) to the instance types defined in the RM.
 
-Where low-level runtimes are focused on the execution of a container within an OS, the more complex/complete high-level container runtimes focus on the general management of container images - getting them from somewhere, unpacking them, and then passing them to the low-level runtime, which then executes the container. These high-level runtimes also include a comprehensive API that other applications (e.g. Kubernetes) can use to interact and manage containers. An example of this type of runtime is containerd, which provides the features described above, before passing off the unpacked container image to runc for execution.
+Where low-level runtimes are used for the execution of a container within an OS, the more complex/complete high-level container runtimes are used for the general management of container images - moving them to where they need to be executed, unpacking them, and then passing them to the low-level runtime, which then executes the container. These high-level runtimes also include a comprehensive API that other applications (e.g. Kubernetes) can use to interact and manage the containers. An example of this type of runtime is containerd, which provides the features described above, before passing off the unpacked container image to runc for execution.
```

Comment:
@bfcohen w.r.t. the following sentence, "An example of this type of runtime is containerd, which provides the features described above, before passing off the unpacked container image to runc for execution."
1. there is something missing because the following is obviously incorrect, "An example of this type of runtime is containerd before passing off the unpacked container image to runc for execution."
2. "passing off the unpacked container image to runc for execution" is also described above.

Reply:
Not sure how to solve this.